lib/driver.js

```diff
@@ -463,6 +458,24 @@ class EspressoDriver extends BaseDriver {
     await this.addDeviceInfoToCaps();
   }
 
+  async setWindowAnimationState(isEnabled) {
+    const isAnimationOn = await this.adb.isAnimationOn();
+    const shouldDisableAnimation = !isEnabled && isAnimationOn;
+    const shouldEnableAnimation = isEnabled && !isAnimationOn;
+
+    if (shouldDisableAnimation) {
+      this.log.debug('Disabling window animation as "disableWindowAnimation" capability is set to true');
```

Comment:
it might not be necessarily set to true (e.g. if the default value is used)

Reply:
updated the message